bitnami/etcd/README.md

```diff
@@ -0,0 +1,178 @@
+| `auth.client.secureTransport`      | Switch to encrypt client communication using TLS certificates                                              | `false`                                                            |
+| `auth.client.useAutoTLS`           | Switch to automatically create the TLS certificates                                                        | `false`                                                            |
+| `auth.client.enableAuthentication` | Switch to enable host authentication using TLS certificates. Requires existing secret.                     | `secret`                                                           |
+| `auth.client.existingSecret`       | Name of the existing secret containing cert files for client communication.                                | `nil`                                                              |
+| `auth.peer.secureTransport`        | Switch to encrypt peer communication using TLS certificates                                                | `false`                                                            |
+| `auth.peer.useAutoTLS`             | Switch to automatically create the TLS certificates                                                        | `false`                                                            |
+| `auth.peer.enableAuthentication`   | Switch to enable host authentication using TLS certificates. Requires existing secret.                     | `false`                                                            |
+| `auth.peer.existingSecret`         | Name of the existing secret containing cert files for peer communication.                                  | `nil`                                                              |
+| `securityContext.enabled`          | Enable security context                                                                                    | `true`                                                             |
+| `securityContext.fsGroup`          | Group ID for the container                                                                                 | `1001`                                                             |
+| `securityContext.runAsUser`        | User ID for the container                                                                                  | `1001`                                                             |
+| `service.type`                     | Kubernetes Service type                                                                                    | `ClusterIP`                                                        |
+| `service.port`                     | etcd client port                                                                                           | `2379`                                                             |
```

Comment:
Shouldn't we include a `service.nodePorts` group of parameters with `service.nodePorts.client` and `service.nodePorts.peer`. This way, the user can configure what node ports to use when setting `service.type` to **NodePort**

Reply:
done